k-distribution/k-tutorial/1_basic/14_evaluation_order/README.md

```diff
@@ -0,0 +1,297 @@
+simply annotating each expression production with information about how
+it is to be evaluated instead. We can do this with the `seqstrict` attribute.
+
+Consider the following definition, once again equivalent to those above
+(`lesson-14-c.k`):
+
+```{.k .alias}
+module LESSON-14-C-SYNTAX
+  imports UNSIGNED-INT-SYNTAX
+  imports BOOL-SYNTAX
+
+  syntax Exp ::= Int
+               | Bool
+               > left: Exp "+" Exp [seqstrict(exp; 1, 2)]
+               > left: Exp "&&" Exp [seqstrict(exp; 1, 2)]
+endmodule
+
+module LESSON-14-C
+  imports LESSON-14-C-SYNTAX
+  imports INT
+  imports BOOL
+
+  rule <k> I1:Int + I2:Int => I1 +Int I2 ...</k>
+  rule <k> B1:Bool && B2:Bool => B1 andBool B2 ...</k>
+
+  context alias [exp]: <k> HERE ...</k>
+
+  syntax Bool ::= isKResult(K) [function, symbol]
+  rule isKResult(_:Int) => true
+  rule isKResult(_:Bool) => true
+  rule isKResult(_) => false [owise]
+endmodule
+```
+
+This definition has two important changes from the one above. The first is
+that the individual `context` sentences have been removed and have been
+replaced with a single `context alias` sentence. You may notice that this
+sentence begins with an identifier in square brackets followed by a colon. This
+syntax is a way of naming individual sentences in K for reference by the tool
+or by other sentences. The context alias sentence also has a special variable
+`HERE`.
+
+The second is that the productions in `LESSON-14-C-SYNTAX` have been given a
+`seqstrict` attribute. The value of this attribute has two parts. The first
+is the name of a context alias sentence. The second is a comma-separated list
+of integers. Each integer represents an index of a non-terminal in the
+production, counting from 1. For each integer present, the compiler implicitly
+generates a new `context` sentence according to the following rules:
+
+1. The compiler starts by looking for the `context alias` sentence named. If
+there is more than one, then one `context` sentence is created per
+`context alias` sentence with that name.
+2. For each `context` created, the variable `HERE` in the context alias is
+substituted with an instance of the production the `seqstrict` attribute is
+attached to. Each child of that production is a variable. The non-terminal
+indicated by the integer offset of the `seqstrict` attribute is given the name
+`HOLE`.
+3. For each integer offset prior in the list to the one currently being
+processed, the predicate `isKResult(E)` is conjuncted together and included
+as a side condition, where `E` is the child of the production term with that
+offset, starting from 1. For example, if the attribute lists `1, 2`, then
+the rule generated for the `2` will include `isKResult(E1)` where `E1` is the
+first child of the production.
+
+As you can see if you work through the process, the above code will ultimately
+generate the same contexts present in `LESSON-14-B`. Finally, note that there
```

Comment:
I would suggest putting the last sentence in it's own paragraph, so it's clear it is introducing the next two paragraphs.